src/commands/component.js

```diff
@@ -89,11 +97,36 @@ class ComponentCommand extends AbstractCommand {
    * @return {Promise}
    * @private
    */
-  _deleteComponent(name) {
+  _saveComponent(name) {
+    const configPath = this._getConfigPath(name)
+    if (configPath) {    
+      const tmpPath = homePath(jitPath);
+      const arch = (new Downloader()).getOsArch();
+      const componentBinPath = `${commandsPath}/../../bin/${arch}`
+      this.logger.info(`Done for terrahub component: '${name}'`);
+      return childProcess.execSync(`${componentBinPath}/component ${tmpPath} ${configPath} ${name}`, { encoding: 'utf8' });
```

Comment:
Please, use `exec` from `child-process-promise` package instead of execSync

Also, this construction can be simplified like:
```
if (!configPath) {
  return Promise.resolve();
}
const tmpPath = homePath(jitPath);
const arch = (new Downloader()).getOsArch();
const componentBinPath = `${commandsPath}/../../bin/${arch}`
this.logger.info(`Done for terrahub component: '${name}'`);

return exec(`${componentBinPath}/component ${tmpPath} ${configPath} ${name}`, { encoding: 'utf8' });
```